conf/messages

```diff
@@ -1708,3 +1708,20 @@ we-only-support-gg.p1=The Government Gateway user ID you signed in with cannot b
 we-only-support-gg.signIn=Sign in if you already have an account
 we-only-support-gg.signIn.p1=<a href="{0}" class="govuk-link">Sign in</a> with the same Government Gateway user ID you used to create your Capital Gains Tax on UK property account.
 we-only-support-gg.title=Report and pay Capital Gains Tax on UK property
+
+
+#===================================================
+# DATE FORMAT
+#===================================================
+month.January=January
```

Comment:
I thought the plan was to change it to month.1=... so that it works for both English and Welsh?

Reply:
the date format was already "11 January 2024" in the frontend. I would need to first parse the date string as a date before being able to get the month value. I know it looks a little odd but this is the easiest way we can do it.